src/librustc_mir/interpret/eval_context.rs

```diff
@@ -207,6 +211,7 @@ impl<'a, 'mir, 'tcx: 'mir, M: Machine<'a, 'mir, 'tcx>> EvalContext<'a, 'mir, 'tc
             param_env,
             memory: Memory::new(tcx, memory_data),
             stack: Vec::new(),
+            vtables: FxHashMap(),
```

Comment:
I think we are supposed to use `FxHashMap::default()` these days.

Reply:
done